projects_test.go

```diff
@@ -1,52 +1,176 @@
 package gitlab
 
 import (
+	"fmt"
+	"net/http"
+	"reflect"
 	"testing"
 )
 
-func TestListAllProjects(t *testing.T) {
-	ts, client := Stub("stubs/projects/index.json")
-	defer ts.Close()
+func TestListProjects(t *testing.T) {
+	setup()
+	defer teardown()
 
-	opt := &ListProjectsOptions{}
-	projects, _, err := client.Projects.ListAllProjects(opt)
+	mux.HandleFunc("/projects", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ah... Here you are using the `mux` I commented about earlier :wink: In that case I would say this one also needs to be returned by the `setup()` func...